source/jquery.flot.hover.js

```diff
@@ -246,6 +250,13 @@ the tooltip from webcharts).
 
         function processRawData() {
             triggerCleanupEvent();
+            var series = plot.getData();
+            if (lastMouseMoveEvent !== undefined) {
+                triggerClickHoverEvent("plothover", lastMouseMoveEvent,
```

Comment:
Instead of copy-pasting the `onMouseMove` call, these should both use a common helper.

Reply:
Fair enough.